project/cmake/scripts/common/HandleDepends.cmake

```diff
@@ -161,6 +161,10 @@ function(add_addon_depends addon searchpath)
                                   PATCH_COMMAND ${PATCH_COMMAND}
                                   "${INSTALL_COMMAND}")
 
+        if(CMAKE_VERSION VERSION_GREATER 3.5)
+          list(APPEND EXTERNALPROJECT_SETUP GIT_SHALLOW 1)
```

Comment:
@fetzerch this conflicts with https://github.com/xbmc/xbmc/blob/a18701e/cmake/addons/depends/README#L17-L18
`<git-revision>` can't be a commit which is not the last commit for a branch